src/Bridge/IBridge.cpp

```diff
@@ -116,6 +119,9 @@ void IBridge::defineOptions(Poco::Util::OptionSet & options)
     options.addOption(
         Poco::Util::Option("stderr-path", "", "stderr log path, default console").argument("stderr-path").binding("logger.stderr"));
 
+    options.addOption(
+        Poco::Util::Option("libraries-path", "", "A colon-separated (:) lists of paths from where we allow to load libraries. Subdirectories are also included. For security, this path should not be accessible for writing by user-controlled applications.").argument("libraries-path").binding("libraries-path"));
```

Comment:
Note about no write access is our recommendation, but it may sound that we enforce it in the code. We can rephrase it a bit:

```suggestion
        Poco::Util::Option("libraries-path", "", "A colon-separated (:) lists of paths from where we allow to load libraries. Subdirectories are also included. To prevent security risks, ensure this path is strictly read-only and not writable by user-controlled applications.").argument("libraries-path").binding("libraries-path"));
```